src/game.cpp

```diff
@@ -13622,11 +13622,29 @@ void intro()
     }
     werase(tmp);
 
-#if !(defined _WIN32 || defined WINDOWS)
+/*                      *** NOTE ***
+ * Not all locale are equal! Gentoo Linux, for instance,
+ * reports ${LANG} for UTF-8 as "en_US.utf8"! Be aware!
+ */
+#if !(defined _WIN32 || defined WINDOWS || defined TILES)
     // Check if locale is has UTF-8 encoding
     char *locale = setlocale(LC_ALL, NULL);
     if (locale != NULL) {
-        if (strstr(locale, "UTF-8") == NULL) {
+        // convert all to uppercase, no need to retain originals
+        for(size_t i = 0; i < strlen(locale); ++i)
+            locale[i] = toupper(locale[i]);
```

Comment:
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x0000000000a5a6f9 in intro () at src/game.cpp:13635
13635               locale[i] = toupper(locale[i]);
(gdb) bt
#0  0x0000000000a5a6f9 in intro () at src/game.cpp:13635
#1  0x00000000009f172f in game::init_ui (this=0x7f5ed94bb010) at src/game.cpp:274
#2  0x0000000000ab362a in main (argc=0, argv=0x7fff5638bcf0) at src/main.cpp:414